akka-persistence-typed/src/main/scala/akka/persistence/typed/javadsl/CommandHandler.scala

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+package akka.persistence.typed.javadsl
+
+import akka.actor.typed.javadsl.ActorContext
+import akka.annotation.InternalApi
+import akka.japi.pf.FI
+import akka.persistence.typed.scaladsl.PersistentBehaviors.EffectImpl
+import akka.util.OptionVal
+
+/**
+ * SAM for reacting on commands
```

Comment:
isn't FunctionalInterface the new official name?

Reply:
Fixed